### ISSUE TITLE
Missing response header

### ISSUE THREAD

**Issue by mirkoperillo** [mirkoperillo/resting#31]
GET www.google.com 
a `set-Cookie` response header is expected but it isn't shown.

**Comment by mirkoperillo** [comment]
confirmed

**Comment by alexmario74** [comment]
The issue is connected to the security framework on Firefox.

The `XMLHTTPRequest` and also the `fetch` are not allowed to access some data about the request like the cookies.

If you have a look to the developer tools in the network tab, you're going to see the request performed and the headers.

This time the missing field `set-cookie` is present.

But in your response from the jQuery call is not.

To by-pass this security issue, you should work on the webRequest intercepting the requests performed by `resting` and save the original response headers.

First of all in your manifest file you should add the permission

    ...
      "permissions": [
        "<all_urls>",
        "webRequest"
      ]
     ...

Then you should write an interceptor attaching an event listener to a webRequest event like `onHeadersReceived`:

```
chrome.tabs.getCurrent(currentTab =>
  browser.webRequest.onHeadersReceived.addListener(
    request => {
      // save headers here
      console.log('response headers:', request.responseHeaders);
    },
    { urls: ['<all_urls>'], tabId: currentTab.id },
    ['responseHeaders'],
  ));
```

I have a workig fix I'm going to send it as a pull request.

**Comment by mirkoperillo** [comment]
@alexmario74 thank you for the clear explanation and contribution.

I'll merge it ASAP

**Comment by mirkoperillo** [comment]
Closed  by PR #61
